tutorials/serverless-grafana-with-iap/code/main.tf

```diff
@@ -0,0 +1,282 @@
+locals {
+  apis = ["iam.googleapis.com", "compute.googleapis.com", "run.googleapis.com", "apigateway.googleapis.com", "servicemanagement.googleapis.com", "servicecontrol.googleapis.com", "iap.googleapis.com", "sql-component.googleapis.com", "cloudapis.googleapis.com", "sqladmin.googleapis.com", "secretmanager.googleapis.com"]
+}
+
+data "google_project" "project" {
+  project_id = var.project_id
+}
+
+resource "google_project_service" "project" {
+  for_each = toset(local.apis)
+  project = data.google_project.project.project_id
+  service = each.key
+  disable_on_destroy = false
+}
+
+data "google_compute_default_service_account" "default" {
```

Comment:
best practice to not use default compute engine SA - can we create a new SA with limited privileges?
Maybe not necessary for a Community tutorial...

Reply:
@cgrotz what do you think?

Reply:
Sure, let's add that